executor/table_reader.go

```diff
@@ -32,6 +35,20 @@ import (
 // make sure `TableReaderExecutor` implements `Executor`.
 var _ Executor = &TableReaderExecutor{}
 
+// selectResult is used to hack distsql.SelectWithRuntimeStats safely for testing.
```

Comment:
What about moving it to `_test` file directly?

Reply:
It will let these executors like `TableReader` involve something in `_test` file, which is weird.